mods/common/languages/en.ftl

```diff
@@ -717,3 +717,30 @@ options-time-limit =
         }
 
 notification-time-limit-expired = Time limit has expired.
+
+## EditorActorBrush
+notification-added-actor = Added { $name } ({ $id })
+
+## EditorCopyPasteBrush
+notification-copied-tiles = Copied { $amount } tiles
+
+## EditorDefaultBrush
+notification-removed-actor = Removed { $name } ({ $id })
+notification-removed-resource = Removed { $type }
+
+## EditorResourceBrush
+notification-added-resource =
+    { $amount ->
+       [one] Added { $amount } cell of { $type }
+      *[other] Added { $amount } cells of { $type }
```

Comment:
This doesn't appear to work:
![grafik](https://github.com/OpenRA/OpenRA/assets/7704140/af2b7051-66b3-4417-b209-868ca81830e3)

Reply:
`one` doesn't seem to be working anymore. You have to use `1`. Did it break in our current fluent version?

Reply:
Not just here. Created https://github.com/OpenRA/OpenRA/issues/20882.